analyses/oncoprint-landscape/01-plot-oncoprint.R

```diff
@@ -0,0 +1,237 @@
+# This script displays an oncoprint displaying the landscape across PBTA given
+# the relevant metadata and output MAF files from the snv callers strelka2,
+# mutect2, lancet, and vardict. It addresses issue #6 in the OpenPBTA-analysis
+# github repository.
+#
+# Code adapted from the PPTC PDX Oncoprint Generation repository here:
+# https://github.com/marislab/create-pptc-pdx-oncoprints/tree/master/R
+#
+# Chante Bethell for CCDL 2019 and Jo Lynne Rokita
+#
+# #### USAGE
+# This script is intended to be run via the command line from the top directory
+# of the repository as follows:
+#
+# Rscript 'analyses/oncoprint-landscape/01-plot-oncoprint.R'
+
+#### Set Up --------------------------------------------------------------------
+
+# Install maftools
+if (!("maftools" %in% installed.packages())) {
+  install.packages("BiocManager")
+  BiocManager::install("maftools")
+}
+library(maftools)
+
+# Source the color palette for plots
+source(file.path("util", "oncoplot-palette.R"))
```

Comment:
This should utilize `root_dir` so it's looking in the right directory no matter where this script is called from.